lab1/README.md

```diff
@@ -1,17 +1,22 @@
 In this lab, we will run a simple Chaos Engineering experiment that looks
-at what could happen when the `middle` service is restarted. We are tring to
+at what could happen when the `middle` service is restarted. We are trying to
 understand the impact on the general flow of the application.
 
+Throughout these labs, we will be using `jq` to extract and display snippets of our experiments which you can install from the [jq download page](https://stedolan.github.io/jq/download/)
```

Comment:
Please try to keep to length of 80 characters max :)

If you use vscode, you can add vertical lines to see that limit, as follows in your settings:

```json
{
    "editor.rulers": [
        80,
        120
    ],
}
```

Reply:
All sorted